core/src/main/java/cz/xtf/core/openshift/OpenShift.java

```diff
@@ -1067,7 +1067,14 @@ public boolean deleteConfigMap(ConfigMap configMap) {
 	}
 
 	// Templates
+	private void updateTemplateApiVersion(Template template) {
```

Comment:
Is there matching openshift-client issue opened? I understand this is just workaround for issue in openshift-client, right?